mappings/net/minecraft/item/HorseArmorItem.mapping

```diff
@@ -1,9 +1,9 @@
 CLASS bbv net/minecraft/item/HorseArmorItem
 	FIELD a bonus I
-	FIELD b texture Ljava/lang/String;
+	FIELD b entityTexture Ljava/lang/String;
 	METHOD <init> (ILjava/lang/String;Lbbw$a;)V
 		ARG 1 bonus
 		ARG 2 name
 		ARG 3 settings
-	METHOD d getHorseArmorTexture ()Lqs;
+	METHOD d getEntityTexture ()Lqs;
```

Comment:
I feel like this could be more descriptive. Maybe emphasize that it is an "in world" texture ? Or that it is for when it is worn.

Reply:
Resolved through brief discussion in Discord. It represents the entity texture of the horse armour and is reasonably descriptive in context. The real concern here is why the hell Mojang store their entity textures in an item class, set up via the constructor... Dammit, Mojang.